### DISCUSSION TITLE
Error using imakeidx using latest TexLIve and lwarp

### DISCUSSION THREAD

**Discussion by dciomics** [bdtc/lwarp#26] [Q&A] [answered]
I upgraded TexLive and am now getting this error generating the lwarp html:

! Undefined control sequence.
\GenericError  ...                                
                                                    #4  \errhelp \@err@     ...
l.26 ...tem accuracy, \hyperindexref{\textbf{196}}

This was working prior to the upgrade, but I may not have had indexing set up properly before. Looking for advice in tracking this down...

**Comment by bdtc** [comment]
There were some changes to the label system a year or so ago.  What is your current version of TeXLive and of lwarp?

If upgrading lwarp to v0.916 does not solve the problem, perhaps you could create a minimal working example file to show the error.

**Reply by dciomics** [reply]
Brian, here are my version numbers:
```
lwarpmk: v0.916  Automated make for the LaTeX Lwarp package.
This is XeTeX, Version 3.141592653-2.6-0.999996 (TeX Live 2024) (preloaded format=xelatex) restricted \write18 enabled.
LaTeX2e <2023-11-01> patch level 1
```

I'll try to create a minimal example that triggers the error. That exercise might shed light on why I'm now getting the error. lwarp has been a lifesaver, so I'm eager to squash this issue.

**Reply by bdtc** [reply]
May want to try lualatex for a moment and see if it matters.

**Reply by dciomics** [reply]
Brian, the error is the same using lualatex. Preparing a minimal example that triggers the error is at the top of my task queue...

**Reply by dciomics** [reply, marked as answer]
Brian, In preparing a minimal example, I tracked down that the issue appears to have been triggered by a \paragraph that had a \(\phi\) in its name. Using \paragraph[no phi]{with $\phi$} eliminated the error for reasons I didn't track down. 

Thanks for creating lwarp and for your supportive suggestions!

**Reply by bdtc** [reply]
I've added this to the "look into it" list, hopefully to have a fix in the next release.
